spyder/plugins/editor/utils/folding.py

```diff
@@ -222,6 +226,7 @@ def detect_fold_level(self, prev_block, block):
         """
         text = block.text()
         prev_lvl = TextBlockHelper().get_fold_lvl(prev_block)
+        regex = r"(and|or|'|\+|\-|\*|\^|>>|<<|\*|\*{2}|\||\*|//|/|,|\\|\")$"
```

Comment:
Please add a better name to this variable. I mean `regex` is not very telling. Perhaps `cont_line_regex`?